pkg/feature/initializer.go

```diff
@@ -47,7 +45,6 @@ func (s *FeaturesInitializer) Apply() error {
 func (s *FeaturesInitializer) Delete() error {
 	var cleanupErrors *multierror.Error
 	for i := len(s.Features) - 1; i >= 0; i-- {
-		log.Info("cleanup", "name", s.Features[i].Name)
```

Comment:
Same as above, logging for the sake of seeing it working. Not convinced that is needed.